src/imageuploadengine.js

```diff
@@ -85,19 +72,6 @@ export default class ImageUploadEngine extends Plugin {
 				}
 			}
 		} );
-
-		// Model to view converter for image's `uploadId` attribute.
-		editor.editing.modelToView.on( 'addAttribute:uploadId:image', ( evt, data, consumable ) => {
-			if ( !consumable.consume( data.item, eventNameToConsumableType( evt.name ) ) ) {
-				return;
-			}
-
-			const modelImage = data.item;
-			const viewFigure = editor.editing.mapper.toViewElement( modelImage );
-			const viewImg = viewFigure.getChild( 0 );
-
-			viewImg.setAttribute( 'src', this.placeholder );
-		} );
 	}
```

Comment:
I might overengineer it, but, isn't it that true that without this converter upload will cause a crash? I believe that engine plugin should work fine, without any additional UI plugin. Then UI plugins should be able to overwrite this very basic converter. Note that the same way there is a converter in the basic styles plugin: https://github.com/ckeditor/ckeditor5-basic-styles/blob/bdc76445715866c948544dc19ff4ad2b569cc302/src/boldengine.js.